checkpoint/runner_test.go

```diff
@@ -231,89 +252,68 @@ func asAtxSnapshot(v *types.ActivationTx, cmt *types.ATXID) types.AtxSnapshot {
 	}
 }
 
-func createMesh(t *testing.T, db *sql.Database, miners map[types.NodeID][]*types.ActivationTx, accts []*types.Account) {
-	for _, vatxs := range miners {
-		vatxs = slices.Clone(vatxs)
-		// ATXs are expected to be in reverse epoch order and we want older ATXs
-		// created first so that the nonce can be retrieved from them for
-		// populating the nonce field when creating newer ones
-		slices.Reverse(vatxs)
-		for _, atx := range vatxs {
+func createMesh(t *testing.T, db *sql.Database, miners []miner, accts []*types.Account) {
+	for _, miner := range miners {
```

Comment:
Add `t.Helper()` before this line

Reply:
Probably also should be `testing.TB` instead of `*testing.T` 🙂

Reply:
> Probably also should be `testing.TB` instead of `*testing.T` 🙂

Why? We rather use the latter in most places in the codebase. TBH either is fine for me as long as we are consistent throughout the repo.

Reply:
You also removed `testing.TB` from a few methods in this PR 😉 

I'm advocating for using `testing.TB` because you can use the helper then in both tests and benchmarks and "other places don't do that" imo isn't an argument for not doing that?  😕

Reply:
> You also removed `testing.TB` from a few methods in this PR 😉

I removed them from where they were unused.

> "other places don't do that" imo isn't an argument for not doing that? 😕

I'm not saying that it is :shrug:. I'm saying "let's be consistent".